.gitignore

```diff
@@ -9,4 +9,6 @@ bin
 # logs
 *.log
 
-.idea/
\ No newline at end of file
+.idea/
+
+packages/engine/paima-rest/src/tsoa/routes.ts
```

Comment:
We can't add this to gitingore I think because it will break some other systems that require parsing this file. The real solution would be trying to find out why this file sometimes gets formatted by a tool (possibly prettier) despite being specified in the `.prettierignore`. Running `prettier` on the git repo as expected ignores this file, but it's possible `nx` or some other tool is not respecting this somewhere

Reply:
but how git relates to these systems?

Reply:
it's just a file that is not tracked by the repo if it is ignored

Reply:
why would it break anything? otherwise one should always specify all subfolders or do `git rm` on this file which is not very convenient. if we don't need a file in a repo it's better to ignore it imo if it doesn't break things

Reply:
Many systems and tools look at git for their behavior

Reply:
as discussed on the call i added back this file since it was not tracked by git and removed it from gitignore